huggle/apiquery.cpp

```diff
@@ -417,7 +417,7 @@ void ApiQuery::SetAction(const Action action)
             this->ActionPart = "login";
             this->EnforceLogin = false;
             return;
-        case ClientLogin:
```

Comment:
I think you forgot to renamed it elsewhere because now it doesn't compile at all

Reply:
fixed